### ISSUE TITLE
LZDoom encounters fatal error on loading

### ISSUE THREAD

**Issue by SHIR0B0N** [pa1nki113r/Project_Brutality#947]
![image](https://user-images.githubusercontent.com/108367127/176270362-9a2e972d-ecfc-4b85-93c6-6822ddcd69fa.png)

Sounds like one of the files uses a function that doesn't exist. Crashes on loading on LZDoom.

Tried with GZDoom 4.8 and it also crashes but only after selecting difficulty.

Any ideas?

**Comment by JMartinez9820** [comment]
@SHIR0B0N 
https://github.com/drdteam/gzdoom-gles/releases/tag/gles4.6.1
Try using this version of GZDoom

**Comment by JMartinez9820** [comment]
This one is what most people recommend along with 4.6.0b gles because it performs better. If this doesn't work then I would suggest posting your PC specs or the crash you have with 4.8.0

**Comment by SHIR0B0N** [comment]
Huh. GZDoom 4.6.1 works fine and dandy but not 4.8 or LZDoom.

Specs on my PC if ever you investigate this crash:

CPU:           Intel Core i7-9700K (Coffee Lake-S, R0)
               3600 MHz (36.00x100.0) @ 4900 MHz (49.00x100.0)
Motherboard:   ASUS PRIME Z390-A
BIOS:          1302, 09/02/2019
Chipset:       Intel Z390 (Cannon Lake-H)
Memory:        16384 MBytes @ 1600 MHz, 16-18-18-36
               - 8192 MB PC25600 DDR4 SDRAM - Crucial Technology BLS8G4D32AESCK.M8F
               - 8192 MB PC25600 DDR4 SDRAM - Crucial Technology BLS8G4D32AESCK.M8F
Graphics:      GIGABYTE RTX 2070 Super Gaming OC White (GV-N207SGAMINGOC WHITE-8GD)
               NVIDIA GeForce RTX 2070 Super, 8192 MB GDDR6 SDRAM
Drive:         WDC WD15EARX-00PASB0, 1465.1 GB, Serial ATA 6Gb/s @ 6Gb/s
Drive:         C400-MTFDDAC064MAM, 62.5 GB, Serial ATA 6Gb/s @ 6Gb/s
Drive:         Samsung SSD 850 EVO 250GB, 244.2 GB, Serial ATA 6Gb/s @ 6Gb/s
Drive:         ST2000DM001-1ER164, 1953.5 GB, Serial ATA 6Gb/s @ 6Gb/s
Sound:         Intel Cannon Lake-H/S - cAVS (Audio, Voice, Speech) [B0]
Sound:         NVIDIA TU104 - High Definition Audio Controller
Network:       Intel Ethernet Connection I219-V
OS:            Microsoft Windows 10 Professional (x64) Build 18363.1256 (1909/November 2019 Update)


Thanks for the speedy response.

**Comment by JMartinez9820** [comment]
Personally I don't see this fixed anytime soon unfortunately so I put the won't fix label on this. If you can screenshot the crash with GZDoom 4.8.0, although I'm sure that you might also get it on 4.8.1

**Comment by SHIR0B0N** [comment]
I can't screenshot it because the computer freezes and all I get is a black screen with the music going on, unfortunately. 

And even then, it seems to work half of the time, sometimes it'll load, sometimes it'll crash. Mem leak maybe? 

Anyway it's running super on 4.6.1 gles. No crashes there.

**Comment by SHIR0B0N** [comment]
I think it's even more embarrassing that I didn't realize I had an outdated version of LZDoom. It works fine on 3.88b. 

yikes
